src/queries/role.ts

```diff
@@ -12,13 +12,22 @@ import type {
   QueryAllByQuery,
   QueryByQuery,
 } from './makeQueries';
+import { matchAccessibilityState } from './a11yState';
 
 type ByRoleOptions = {
   name?: TextMatch;
 } & AccessibilityState;
 
 type AccessibilityStateKey = keyof AccessibilityState;
 
+const accessibilityStateKeys: AccessibilityStateKey[] = [
```

Comment:
We could consider exporting that directly from `./a11yState`, or compute it from there

Reply:
Ok, makes sense to move it, but I think that `.accessiblity.ts` would probably be a better place